mdflynn/refactor-tractor-fitlit
# Compile and prep for deployment to github pages

## What is the change?
This adds the necessary changes to see the website deployed in GitHub Pages.

## What does it fix?
No fix this PR.

## Is this a fix or a feature?
This is a feature.

## Where should the reviewer start?
Review the entire styles.css and main.bundle.js to see compiled files.

## How should this be tested?
Navigate to https://mdflynn.github.io/refactor-tractor-fitlit/ to see deployed site.
